modules/installation-configuration-parameters.adoc

```diff
@@ -1778,6 +1778,10 @@ a|Multiple IP addresses
 |If you define multiple failure domains for your cluster, you must attach the tag to each vCenter cluster. To define a zone, use a tag from the `openshift-zone` tag category. For a single vSphere datacenter environment, you do not need to attach a tag, but you must enter an alphanumeric value, such as `cluster`, for the parameter.
 |`String`
 
+|`platform.vsphere.failureDomains.template`
+|Specify the absolute path to a pre-existing {op-system} image template or virtual machine. The installation program can use the image template or virtual machine to quickly install {op-system-first:} on vSphere hosts. Consider using this parameter as an alternative to uploading an {op-system} image on vSphere hosts.
```

Comment:
@jcpowermac . Parameter can be used for IPI or UPI? Reason is that I only see the detailed "RHCOS OVA image" steps in the UPI docs. See [Installing RHCOS and starting the OpenShift Container Platform bootstrap process](https://docs.openshift.com/container-platform/4.13/installing/installing_vsphere/installing-vsphere.html#installation-vsphere-machines_installing-vsphere).

Reply:
Do machine pool config parameters come into play with the "template" parameter? Reason I see a statement about restricted networks "must" pursue the OVA image route. See [Optional VMware vSphere machine pool configuration parameters](https://docs.openshift.com/container-platform/4.13/installing/installing_vsphere/installing-vsphere.html#installation-configuration-parameters-optional-vsphere_installing-vsphere).

![Screenshot from 2023-07-14 12-02-59](https://github.com/openshift/openshift-docs/assets/57954076/fb7359af-d63c-40fc-a119-dbf0417e26b5)

Reply:
- Only really makes sense in IPI.
- Also some message that the RHCOS major version (e.g. 4.13) needs to be the same as the version being installed. Though there are checks in the installer to make sure. https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/latest/
- I would remove the **must**.  Someone could still use that, I have no idea why they would want to though, using the existing template is way easier.

Reply:
Thanks, @jcpowermac . I updated the `platform.vsphere.clusterOSImage` to state:

`platform.vsphere.clusterOSImage`

> The location from which the installation program downloads the RHCOS image. Before setting a path value for this parameter, ensure that the RHCOS image's version matches the version of RHCOS that you installed on your OpenShift Container Platform cluster.

WDYT?

Reply:
I updated the `platform.vsphere.failureDomains.template` entry to now state:

> Specify the absolute path to a pre-existing RHCOS image template or virtual machine. The installation program can use the image template or virtual machine to quickly install RHCOS on vSphere hosts. Consider using this parameter as an alternative to uploading an RHCOS image on vSphere hosts. **The parameter is available for use only on installer-provisioned infrastructure.**

Emphasis on IPI. Filtering out this parameter in a table that is shared among IPI and UPI docs is troublesome. The statement is the best working solution.